sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -43,7 +43,6 @@
     from ._lease import BlobLeaseClient
     from ._models import (
         ContainerProperties,
-        BlobProperties,
```

Comment:
Point for whole review, did we decided we wanted to add a deprecation warning for people using `BlobProperties`?

Reply:
Sure, I think that would be a good idea! This is what Anna suggested on the previous iteration of this PR:
```
import warnings
warnings.warn(
    "The use of a 'BlobProperties' instance is deprecated. Please use str instead.",
    DeprecationWarning
)
```